eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelArtifactDeployedEvent (ArtD)
+
+The EiffelArtifactDeployedEvent declares that a software artifact (declared by EiffelArtifactCreatedEvent) has been deployed into an Environemnt (declared by EiffelEnvironmentDefinedEvent) and is consequently available for use.
```

Comment:
need clarification please;
In the simple.json for the EiffelArtifactDeployedEvent.md I have created two links.
  "links": [
    {
      "type": "ENVIRONMENT",
      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0",
    },
    {
      "type": "ARTIFACT",
      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee3",
    }
  ]
}
  ]
}

and/or do you want me to clarify in the description ?
i.e  
from 
software artifact (declared by EiffelArtifactCreatedEvent)
to
software artifact (linked by EiffelArtifactCreatedEvent) ?